src/tools/illink/src/ILLink.Shared/DynamicallyAccessedMemberTypesEx.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.Diagnostics.CodeAnalysis
+{
+	public static class DynamicallyAccessedMemberTypesEx
```

Comment:
Just curious why we need to add this? Is it that we don't have live flow of bits between the libraries and the trimmer code base here? If that is the case, migh be worth adding a comment so that eventually we delete this.

Reply:
ILLink is needed to build the framework so we can't build it with the live framework.